bin/load-mocks

```diff
@@ -541,20 +549,22 @@ def main(num_events=1, extra_events=False):
                     environment=ENVIRONMENTS.next(),
                     user=generate_user(),
                 )
-
-            Commit.objects.get_or_create(
-                organization_id=org.id,
-                repository_id=repo[0].id,
-                key=sha1(uuid4().hex).hexdigest(),
-                defaults={
-                    'author': CommitAuthor.objects.get_or_create(
-                        organization_id=org.id,
-                        email=user.email,
-                        defaults={'name': user.name}
-                    )[0],
-                    'message': 'Ooops!\nFixes {}'.format(event5.group.qualified_short_id),
-                },
-            )[0]
+            try:
+                Commit.objects.get_or_create(
+                    organization_id=org.id,
+                    repository_id=repo[0].id,
+                    key=sha1(uuid4().hex).hexdigest(),
+                    defaults={
+                        'author': CommitAuthor.objects.get_or_create(
+                            organization_id=org.id,
+                            email=user.email,
+                            defaults={'name': user.name}
+                        )[0],
+                        'message': 'Ooops!\nFixes {}'.format(event5.group.qualified_short_id),
+                    },
+                )[0]
+            except AttributeError:
```

Comment:
same here -- i'm just worried that the broadness of `AttributeError` will hide other bugs. is there a way you can check a more specific case?